### DISCUSSION TITLE
[FEATURE] Dawarich as Home Assistant addon

### DISCUSSION THREAD

**Discussion by Frankster-NL** [Freika/dawarich#413] [Ideas]
As Home Assistant (or HAOS) does not allow normal Docker containers to be run on the instance it would be great if someone would turn it into a home assistant addon for easy installation and maintaining a 'healthy' home assistant installation.

**Comment by tabacha** [comment]
The Application should store points if dawarich is temporary not reachable.

**Comment by thomdev-j** [comment]
I built this! You can find it here: https://github.com/thomdev-j/homeassistant-app-dawarich
Based on Dawarich `v1.3.3`.

It runs a full Dawarich instance as a native Home Assistant app - no separate Docker setup needed. Everything is bundled in a single container.

To install, just add the repo as a custom repository in your HA app store and install Dawarich.

**Reply by Frankster-NL** [reply]
You’re a rockstar!
